### PR TITLE
Add iOS-compatible regex for too common revolving adservers

### PR DESCRIPTION
**Pull request by Yuki2718**
### To avoid invalid pull requests, please check and confirm following terms

- [x] This is not an ad/bug report;
- [x] My code follows the [guidelines](https://github.com/AdguardTeam/AdguardFilters/blob/master/CONTRIBUTING.md) and [syntax](https://kb.adguard.com/general/how-to-create-your-own-ad-filters) of this project;
- [x] I have performed a self-review of my own changes;
- [x] My changes do not break web sites, apps and files structure.

## What problem does the pull request fix?

Those regex
```
/^https?:\/\/(?:www\.)?[a-z0-9-]{5,}\.(?:com|bid|online|top|club)\/[0-9a-f]{32}\/invoke\.js/$script,third-party
/^https?:\/\/(?:www\.|[a-z0-9]{7,10}\.)?[a-z0-9-]{5,}\.(?:com|bid|link|live|online|top|club)\/(?:[a-z0-9]{2}\/){2,3}[a-f0-9]{32}\.js/$script,xmlhttprequest,third-party
```
have been catching so many revolving adservers for years, but are not compatible to CB. The result is iOS users see a lot more ads/popup by them than other platforms.

### If the problem does not fall under any category that is listed here, please write a comment below in corresponding section

- [x] Missed ads or ad leftovers;
- [ ] Website or app doesn't work properly;
- [ ] AdGuard gets detected on a website;
- [ ] Missed analytics or tracker;
- [ ] Social media buttons — share, like, tweet, etc;
- [ ] Annoyances — pop-ups, cookie warnings, etc;
- [x] Filters maintenance.

## What issue is being fixed?

Many, I'm sure you know.

### Enter the issue address

An example of the first regex is currently seen on `www.tvsubtitles.net​` and the second `avgle.com` on my side, but they may change.

#### If possible, a screenshot of a page or application should not be cropped too much. Otherwise, it is not always clear where the element is located

0. **DO NOT** upload screenshots with sexually explicit material on GitHub directly.\
 Instead, upload it to a third-party image hosting and post the link to it without preview (!) here.\
 Also, mention if the link leads to NSFW content;

1. Add screenshots of the problem. You can drag and drop images or paste them from clipboard

    Use `<details> </details>` tag to hide screenshots under the spoiler;

2. Describe the issue in detail until it is absolutely clear from the screenshot what the problem is

    You can also indicate any other information that you think the developers should know.

**Warning:** Please remove personal information before uploading screenshots!

1. Your comment

I don't know whether `(?:www\.)?` works on CB so didn't add. Tell me if it works. I concentrated on covering as much targets by as simple regex as possible.

- Almost 100% of them now use .com domain
- There are certainly cases starting from www or pl******* but rather rare
- `(?:[a-z0-9]{2}\/)` repeated only two times is rare

Also the original regex put in specific.txt seems to be only historical reason, they are generic pattern so moved to general_url.txt.

</details>

### Terms

- [x] By submitting this issue, I agree that pull request does not contain private info and all conditions are met

### CODE REVIEW

File: EnglishFilter/sections/general_url.txt [hunk 1]
```diff
@@ -53,6 +53,17 @@
 ! Sample URL: ||js.passaro-de-fogo.biz/t/113/750/a1113750.js
 /\/t\/[0-9]{3}\/[0-9]{3}\/a[0-9]{4,9}\.js$/$script
 !
+! generic regex
```
Comment (Alex-302) [inline EnglishFilter/sections/general_url.txt:56]: It is for Popads, right?

Comment (Yuki2718) [reply EnglishFilter/sections/general_url.txt:56]: TBH I don't know. To me PopAds is those caught by `("abort-on-property-write", "_pop")` and their URL pattern is different.

Comment (Alex-302) [reply EnglishFilter/sections/general_url.txt:56]: :) Ok, please show few urls, matched by these regexes.

Comment (Yuki2718) [reply EnglishFilter/sections/general_url.txt:56]:
```
https://regex101.com/r/tl281M/1
https://regex101.com/r/bZA6Vu/1
```

Comment (Alex-302) [reply EnglishFilter/sections/general_url.txt:56]: Can't open that scripts:)
Do you have their saved content? Just want to look.

Comment (Yuki2718) [reply EnglishFilter/sections/general_url.txt:56]: Saving the page as html doesn't work, maybe try another browser? @zloyden @AdamWr @Sergey-Lyapin Can any of you open the links?

Comment (Alex-302) [reply EnglishFilter/sections/general_url.txt:56]: I mean I can't open ad scripts from tests) regex101.com works fine.

Comment (Yuki2718) [reply EnglishFilter/sections/general_url.txt:56]: What do you mean by saying can't open? Do you see my Unit tests? if you see, then you can open those script URLs by copy & paste e.g. `https://alterupholdcinema.com/51/5f/e0/515fe0778757eae5228770cf1427e721.js`

Comment (Alex-302) [reply EnglishFilter/sections/general_url.txt:56]: I suggest to place new rules under directive, without removing current rules
```
!#if (adguard_app_ios || adguard_ext_safari)

!#endif
```

Comment (Yuki2718) [reply EnglishFilter/sections/general_url.txt:56]: Do you mean put them back to specific.txt?

Comment (zloyden) [reply EnglishFilter/sections/general_url.txt:56]: @Yuki2718 hi! I've looked for your conservation and I guess what you should do:
1. Revert the old rules
2. Move new rules that have `!+ PLATFORM(ios, ext_safari, ext_android_cb)` to the `specific.txt`, but without hints, use the directive.

Comment (Yuki2718) [reply EnglishFilter/sections/general_url.txt]: Done.

**Review by zloyden — APPROVED** [review]

**Pull request merged**